lib/mockRequest.js

```diff
@@ -191,6 +191,47 @@ function createRequest(options) {
         return Accepts.type(types);
     };
 
+    /**
+     * Check if the given `encoding`s are accepted.
+     *
+     * @param {String} ...encoding
+     * @return {String|Array}
+     * @public
+     */
+
+    mockRequest.acceptsEncodings = function(){
+        var accept = accepts(this);
```

Comment:
Use `mockRequest` instead of `this`
Check how it is done in `mockRequest.accepts`

Reply:
done